### DISCUSSION TITLE
Which model files should I keep?

### DISCUSSION THREAD

**Discussion by geocine** [voicepaw/so-vits-svc-fork#641] [Q&A] [answered]
Should it be the `logs/G_*.pth` or the `logs/D_*pth` ?

**Comment by hataori-p** [comment, marked as answer]
Keep the G model (generator) for the inference only.
If you plan to continue training in the future, keep both.

**Reply by nabeelimran** [reply]
if I have a pre trained model and have G_*.pth file for it, how can I use it to infer

**Reply by hataori-p** [reply]
In svcg select the file as a model, or pass it as a model parameter into infer script.

**Reply by nabeelimran** [reply]
I want to use these models https://huggingface.co/QuickWick/Music-AI-Voices/tree/main

should I put pth and config file to a specific folder first?

**Reply by hataori-p** [reply]
In the GUI it doesn't matter where you place it. Set the G_ model in "Model path" and config.json in "Config path".
